src/Export/VTK.jl

```diff
@@ -1,134 +1,47 @@
-export vtk_nodeset, vtk_cellset
+cell_to_vtkcell(::Type{Line}) = VTKCellTypes.VTK_LINE
+cell_to_vtkcell(::Type{QuadraticLine}) = VTKCellTypes.VTK_QUADRATIC_EDGE
 
-"""
-Creates an unstructured VTK grid from the element topology and coordinates.
-
-```julia
-vtk_grid{dim,T}(filename::AbstractString, coords::Vector{Vec{dim,T}}, topology::Matrix{Int}, celltype::VTKCellTypes.VTKCellType)
-```
-
-**Arguments**
-
-* `filename`: name (or path) of the file when it is saved to disk, eg `filename = "myfile"`, or `filename = "/results/myfile"` to store it in the folder results
-* `coords`: a vector of the node coordinates
-* `topology`: a matrix where each column contains the nodes which connects the element
-* `celltype`: the definition of the celltype in the grid, see [https://github.com/jipolanco/WriteVTK.jl#defining-cells](https://github.com/jipolanco/WriteVTK.jl#defining-cells)
-
-**Results:**
-
-* `::DatasetFile`
-
-**Example:**
-
-```jldoctest
-julia> coords = [Vec{2}((0.0,0.0)), Vec{2}((1.0,0.0)), Vec{2}((1.5,1.5)), Vec{2}((0.0,1.0))];
+cell_to_vtkcell(::Type{Quadrilateral}) = VTKCellTypes.VTK_QUAD
+cell_to_vtkcell(::Type{QuadraticQuadrilateral}) = VTKCellTypes.VTK_BIQUADRATIC_QUAD
+cell_to_vtkcell(::Type{Triangle}) = VTKCellTypes.VTK_TRIANGLE
+cell_to_vtkcell(::Type{QuadraticTriangle}) = VTKCellTypes.VTK_QUADRATIC_TRIANGLE
+cell_to_vtkcell(::Type{Cell{2,8,4}}) = VTKCellTypes.VTK_QUADRATIC_QUAD
 
-julia> topology = [1 2 4; 2 3 4]';
+cell_to_vtkcell(::Type{Hexahedron}) = VTKCellTypes.VTK_HEXAHEDRON
+cell_to_vtkcell(::Type{Tetrahedron}) = VTKCellTypes.VTK_TETRA
+cell_to_vtkcell(::Type{QuadraticTetrahedron}) = VTKCellTypes.VTK_QUADRATIC_TETRA
 
-julia> celltype = VTKCellTypes.VTK_TRIANGLE;
-
-julia> vtkobj = vtk_grid("example", coords, topology, celltype);
-
-julia> vtk_save(vtkobj)
-1-element Array{String,1}:
- "example.vtu"
-```
-
-**Details**
-
-This is a thin wrapper around the function `vtk_grid` from the [`WriteVTK`](https://github.com/jipolanco/WriteVTK.jl) package.
-
-For information how to add cell data and point data to the resulting VTK object as well as how to write it to a file see
-[https://github.com/jipolanco/WriteVTK.jl#generating-an-unstructured-vtk-file](https://github.com/jipolanco/WriteVTK.jl#generating-an-unstructured-vtk-file)
 """
-function vtk_grid{dim,T}(filename::AbstractString, coords::Vector{Vec{dim,T}}, topology::Matrix{Int}, celltype::VTKCellTypes.VTKCellType)
```

Comment:
OK to remove this? It feels superseeded by `vtk_grid(filename, grid::Grid)`, at least I don't use it.